EdFi.Buzz.Installer/eng/Windows/example.configuration.json

```diff
@@ -53,6 +54,6 @@
         "version": "0.1.0",
         "url": "http://localhost:4200",
         "port": 4200,
-        "nginxPort": 4200
+        "nginxPort": 4200		
```

Comment:
Interesting that this was changed to add spaces to the end. Your eslint might need a check?

Reply:
Spaces were removed.